.env.development

```diff
@@ -1,3 +1,4 @@
 # Default admin/district authorization for development
 REACT_APP_DHIS2_BASE_URL="http://localhost:8080"
 REACT_APP_DHIS2_AUTHORIZATION="Basic YWRtaW46ZGlzdHJpY3Q="
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Is this line necessary? I don't have it in my local .env, and the tests run without errors for me.

Reply:
I removed it